### ISSUE TITLE
Failed to parse metric datadog.tracer.flush_duration

### ISSUE THREAD

**Issue by vesquen** [DataDog/datadog-lambda-extension#488]
Hi Team

I wanted to report an error that's being logged very frequently. We're using the latest version v67 (Next-generation Lambda Extension).

`DD_EXTENSION | ERROR | Failed to parse metric datadog.tracer.flush_duration:0.785551|ms|#lang:go,lang_version:go1.23.2,env:<redacted>,_dd.origin:lambda,runtime-id:<redacted>,tracer_version:v1.70.1,service:<redacted>,env:<redacted>,service:<redacted>,version:<redacted>: parse failure: Invalid metric format`

Version of dd-lambda-go: v1.20.0.

Is there anything we can do here in the short term? No error when using `DD_EXTENSION_VERSION: compatibility`.

Thank you, kind regards

**Comment by vesquen** [comment]
Hi again.

I wanted to check if there is a way to disable the extension logs for now or any other workaround. `DD_LOG_LEVEL: OFF` doesn't seem to work and the log volume generated by this is huge.

**Comment by alexgallotta** [comment]
Thanks fore reporting it!
It seems to be indeed just a log issue. The fix https://github.com/DataDog/libdatadog/pull/777 is ready and it will be release as soon as possible

**Comment by velikoff** [comment]
Hi Team,
I just wanted to clarify if the release will fully fix the parsing issue or if it will only stop logging the error?

We’re currently experiencing a similar issue in our AWS Lambda function (using .NET) with Histogram metrics. Here’s an example of the error we’re seeing:
_DD_EXTENSION | ERROR | Failed to parse metric lambda.process_time:359|h|#application:Lambda: parse failure: Invalid metric format_

**Comment by AshveenBansal98** [comment]
Hi, 
Any ETA on this? We are also facing this issue 

`Failed to parse metric datadog.tracer.flush_duration:5.983098|ms|#lang:go,lang_version:go1.23.4,env:dev,_dd.origin:lambda,runtime-id:209549e7-2e19-47f9-8039-a03ad82e5038,tracer_version:v1.68.0,service:events-lambda,env:dev,service:events-lambda: parse failure: Invalid metric format`

**Comment by duncanista** [comment]
Hey, this warnings should have been muted on @alexgallotta's PR in the latest [v68](https://github.com/DataDog/datadog-lambda-extension/releases/tag/v68).

@alexgallotta will update further on @velikoff question soon, he currently is OOO

**Comment by alexgallotta** [comment]
TL;DR
the fix is not yet released, and yes it will not produce that log anymore

Details:
The fix is in dogstatsd lib, in this PR https://github.com/DataDog/libdatadog/pull/777/files

It was not included in v68 since it is using dogstatsd rev `dbc8869c6de4ccd78a770524d250420f397fc7b3`
https://github.com/DataDog/datadog-lambda-extension/blob/v68/bottlecap/Cargo.toml#L18

which points to a branch that does not include that change https://github.com/DataDog/libdatadog/commit/dbc8869c6de4ccd78a770524d250420f397fc7b3

**Comment by alexgallotta** [comment]
The fix is merged and it will be available on v69!

**Comment by astuyve** [comment]
This is out in [v69](https://github.com/DataDog/datadog-lambda-extension/releases/tag/v69)
